samples/Chat/src/app/styles/ChatHeader.styles.ts

```diff
@@ -30,21 +30,21 @@ export const leaveIcon: IIconProps = {
   iconName: 'Leave'
 };
 
-export const panelButtonStyle = mergeStyles({
-  padding: '0px 13px',
-  cursor: 'pointer'
```

Comment:
This CSS change seems unrelated? (cursor styling got dropped?)

Reply:
it is related. No need to force the cursor to pointer as it's done automatically now that we use IconButton component instead of just Icon